script.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+
+sudo dnf update
```

Comment:
You do not need to add sudo on every command. You can run the script with sudo. That should do.